src/lib/config.js

```diff
@@ -101,16 +137,37 @@ function validateLocalEnvConfig () {
   }
 }
 
+function validateNotificationEnv () {
+  // Validate notification signing public keys
+  const notifications = parseNotificationSignEnv()
+
+  if (notifications.must_verify) {
+    const ledgers = parseLedgers()
+    for (let ledgerObj of ledgers) {
+      const uri = ledgerObj.ledger
+      if (notifications.keys[uri] === undefined) {
+        throw new Error(`Missing notification signing keys for ledger: ${uri}`)
+      }
+
+      try {
+        fs.accessSync(notifications.keys[uri], fs.R_OK)
```

Comment:
So the point of this call is to prevent us from having to a `try`/`catch` block when reading the file?